idea_town/experiments/tests.py

```diff
@@ -82,6 +82,8 @@ def test_index(self):
                         "xpi_url": "",
                         "details": [],
                         "contributors": [],
+                        "installation_count": UserInstallation.objects.distinct('user')
```

Comment:
To declutter this line a bit, you might want to assign this to a temp variable (e.e `installation_count`) before the `self.assertJSONEqual`.